### PR TITLE
Add Xfce Terminal

### PR DESCRIPTION
**Pull request by f2404**
## Checklist for submitting a pull request to `Terminals Are Sexy`:

- [x] I have carefully **read and comply** with the [Contributing Guidelines](https://github.com/k4m4/terminals-are-sexy/blob/master/contributing.md) of this repo.
- [x] I have **searched** the [PRs](https://github.com/k4m4/terminals-are-sexy/pulls) of this repo and believe that this is not a duplicate.

 

- **Title**: Xfce Terminal
- **Link**: https://docs.xfce.org/apps/terminal/start
- **Category**: Terminal emulator
- **Description**: A modern terminal emulator primarily for the Xfce desktop environment

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -131,6 +131,7 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
 * [Tilix](https://gnunn1.github.io/tilix-web/) - Advanced GTK3 tiling terminal emulator that follows the Gnome Human Interface Guidelines.
 * [TotalTerminal](https://totalterminal.binaryage.com/) - System-wide terminal available on a hot-key.
 * [Upterm](https://github.com/railsware/upterm) - Terminal emulator for the 21st century.
+* [Xfce Terminal](https://docs.xfce.org/apps/terminal/start) - A modern terminal emulator primarily for the Xfce desktop environment.
```
[the remark this answers is not in the document]
Comment (f2404) [reply readme.md]: Done.

**Comment by f2404** [comment]
It seems we've got one endorsement, are more needed?

**Comment by k4m4** [comment]
Hi @f2404. Please squash your commits into one. Thanks!

**Comment by f2404** [comment]
@k4m4 Done!

**Pull request merged**